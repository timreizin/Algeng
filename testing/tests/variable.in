a=5
a
b=1344
b
c=-424
c
d=(4+2)+38*3*(2+4+(47+23*(3+2)+1))
d
