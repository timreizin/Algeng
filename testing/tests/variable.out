5
1344
-424
47124
